version/version.go

```diff
@@ -12,9 +12,10 @@ const (
 )
 
 var (
-	version           = "2.9.0"
-	managedAPIVersion = "1.22.0"
-	log               = l.NewLoggerWithContext(l.Fields{l.ComponentLogContext: "version"})
+	version                      = "2.9.0"
+	managedAPIVersion            = "1.22.0"
+	multitenantManagedAPIVersion = "1.21.0"
```

Comment:
Hey Carl, is there any real need for this addition? 
I thought that an MT release would be the same as RHOAM release but with a slightly different setup on the bundle level.

Reply:
After discussing this on the main channel, we decided to keep this approach with separate versions for MT and standard RHOAM in order to simplify support for patch releases.